### PR TITLE
Changes to Typescript Quickstart

### PR DESCRIPTION
**Pull request by kdhamric**
Made several changes:

A large number of our quick starts reference the docs in the readme rather than duplicating the steps to run - I changed the Readme to follow this convention and pointed users to it. 

Moved the steps to run the quick start from the Readme into the docs.

Removed material explaining how the tracetest agent connection, collector configuration, and docker compose network are configured so the quick start focuses solely on the use of the NPM package. As I read the quick start initially all the other information distracted me from the goal of showing how to use the NPM library.

### CODE REVIEW

File: docs/docs/tools-and-integrations/typescript.mdx [hunk 1]
```diff
@@ -63,31 +63,50 @@ This is a simple quick-start guide on how to use the Tracetest `@tracetest/clien
 
 **Docker**: Have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed on your machine.
 
-## Installing the `@tracetest/client` NPM Package
+## Run this Quckstart example
```
Comment (jfermi) [inline docs/docs/tools-and-integrations/typescript.mdx:66]:
```suggestion
## Run This Quckstart Example
```

File: docs/docs/tools-and-integrations/typescript.mdx [hunk 2]
```diff
@@ -63,31 +63,50 @@ This is a simple quick-start guide on how to use the Tracetest `@tracetest/clien
...
+cd tracetest/examples/quick-start-typescript
 ```
 
-## Using the Tracetest Client Package
+Follow these instruction to run the included demo app and typescript example:
 
-Once you have installed the `@tracetest/client` package, you can import it and start making use of it as any other library to trigger trace-based tests and run checks against the resulting telemetry data.
+1. Copy the `.env.template` file to `.env`.
+2. Log into the [Tracetest app](https://app.tracetest.io/).
+3. This example is configured to use of the OpenTelemetry Collector. Ensure the environment you will be utlizing to run this example is also configured to use the OpenTelemetry Tracing Backend by clicking on Settings, Tracing Backend, OpenTelemetry, Save.
```
Comment (jfermi) [inline docs/docs/tools-and-integrations/typescript.mdx:81]:
```suggestion
3. This example is configured to use the OpenTelemetry Collector. Ensure the environment you will be utilizing to run this example is also configured to use the OpenTelemetry Tracing Backend by clicking on Settings, Tracing Backend, OpenTelemetry, Save.
```

**Review by jfermi — APPROVED** [review]
Are there places where screenshots would be helpful?

File: docs/docs/tools-and-integrations/typescript.mdx [hunk 3]
```diff
@@ -63,31 +63,50 @@ This is a simple quick-start guide on how to use the Tracetest `@tracetest/clien
 
 **Docker**: Have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed on your machine.
 
-## Installing the `@tracetest/client` NPM Package
+## Run This Quckstart Example
+
+The example below is provided as part of the Tracetest project. You can download and run the example by following these steps:
 
-Installing the `@tracetest/client` NPM Package is as easy as running the following command:
+Clone the Tracetest project and go to the Typescript Quickstart:
```
Comment (adnanrahic) [inline docs/docs/tools-and-integrations/typescript.mdx:70]:
```suggestion
Clone the Tracetest project and go to the TypeScript Quickstart:
```

File: docs/docs/tools-and-integrations/typescript.mdx [hunk 4]
```diff
@@ -63,31 +63,50 @@ This is a simple quick-start guide on how to use the Tracetest `@tracetest/clien
...
 
 ```bash
-npm i @tracetest/client
+git clone https://github.com/kubeshop/tracetest
+cd tracetest/examples/quick-start-typescript
 ```
 
-## Using the Tracetest Client Package
+Follow these instruction to run the included demo app and typescript example:
```
Comment (adnanrahic) [inline docs/docs/tools-and-integrations/typescript.mdx:77]:
```suggestion
Follow these instructions to run the included demo app and TypeScript example:
```

File: docs/docs/tools-and-integrations/typescript.mdx [hunk 5]
```diff
@@ -63,31 +63,50 @@ This is a simple quick-start guide on how to use the Tracetest `@tracetest/clien
...
 
-Once you have installed the `@tracetest/client` package, you can import it and start making use of it as any other library to trigger trace-based tests and run checks against the resulting telemetry data.
+1. Copy the `.env.template` file to `.env`.
+2. Log into the [Tracetest app](https://app.tracetest.io/).
+3. This example is configured to use the OpenTelemetry Collector. Ensure the environment you will be utilizing to run this example is also configured to use the OpenTelemetry Tracing Backend by clicking on Settings, Tracing Backend, OpenTelemetry, Save.
+4. Fill out the [token](https://docs.tracetest.io/concepts/environment-tokens) and [agent API key](https://docs.tracetest.io/concepts/agent) details by editing your .env file. You can find these values in the Settings area for your environment.
+5. Run `docker compose up -d`.
+6. Look for the `tracetest-client` service in Docker and click on it to view the logs. It will show the results from the trace-based tests that are triggered from the index.ts Typescript file.
+7. Follow the links in the log to to view the test runs programmatically created by your Typescript program.
```
Comment (adnanrahic) [inline docs/docs/tools-and-integrations/typescript.mdx:85]:
```suggestion
7. Follow the links in the log to to view the test runs programmatically created by your TypeScript test script.
```

File: docs/docs/tools-and-integrations/typescript.mdx [hunk 6]
```diff
@@ -63,31 +63,50 @@ This is a simple quick-start guide on how to use the Tracetest `@tracetest/clien
...
+
+The sections below breakdown in detail what the example you just ran did and how it works.
```
Comment (adnanrahic) [inline docs/docs/tools-and-integrations/typescript.mdx:87]:
```suggestion
Follow along with the sections below for an in detail breakdown of what the example you just ran did and how it works.
```

File: docs/docs/tools-and-integrations/typescript.mdx [hunk 7]
```diff
@@ -63,31 +63,50 @@ This is a simple quick-start guide on how to use the Tracetest `@tracetest/clien
...
 
 ## Project Structure
 
-The project is built with Docker Compose.
+The quick start Typescript project is built with Docker Compose.
```
Comment (adnanrahic) [inline docs/docs/tools-and-integrations/typescript.mdx:91]:
```suggestion
The quick start TypeScript project is built with Docker Compose.
```

File: docs/docs/tools-and-integrations/typescript.mdx [hunk 8]
```diff
@@ -63,31 +63,50 @@ This is a simple quick-start guide on how to use the Tracetest `@tracetest/clien
...
 
-### Pokeshop Demo App
+The [Pokeshop Demo App](/live-examples/pokeshop/overview) is a complete example of a distributed application using different backend and front-end services. We will be launching it and running tests against it as part of this example.
```
Comment (adnanrahic) [inline docs/docs/tools-and-integrations/typescript.mdx:93]:
```suggestion
The [Pokeshop Demo App](/live-examples/pokeshop/overview) is a complete example of a distributed application using different back-end and front-end services. We will be launching it and running tests against it as part of this example.
```

File: docs/docs/tools-and-integrations/typescript.mdx [hunk 9]
```diff
@@ -63,31 +63,50 @@ This is a simple quick-start guide on how to use the Tracetest `@tracetest/clien
...
 
-The [Pokeshop Demo App](/live-examples/pokeshop/overview) is a complete example of a distributed application using different backend and front-end services, implementation code is written in Typescript.
+The `docker-compose.yaml` file in the root directory of the quick start runs the Pokeshop Demo app, the OpenTelemetry Collector setup, and the [Tracetest Agent](/concepts/agent).
 
-The `docker-compose-yaml` file in the root directory is for the Pokeshop Demo app, the OpenTelemetry setup, and the [Tracetest Agent](/concepts/agent).
+The Typescript Quickstart has two primary files: a Typescript file `definitions.ts` that defines two Tracetest tests, and a Typescript file 'index.ts' that imports these test definitions and uses the "@tracetest/client" NPM package to run them multiple times.
```
Comment (adnanrahic) [inline docs/docs/tools-and-integrations/typescript.mdx:97]:
```suggestion
The TypeScript quick start has two primary files: a Typescript file `definitions.ts` that defines two Tracetest tests, and a Typescript file 'index.ts' that imports these test definitions and uses the "@tracetest/client" NPM package to run them multiple times. 
```

File: docs/docs/tools-and-integrations/typescript.mdx [hunk 10]
```diff
@@ -63,31 +63,50 @@ This is a simple quick-start guide on how to use the Tracetest `@tracetest/clien
...
+The [Pokeshop Demo App](/live-examples/pokeshop/overview) is a complete example of a distributed application using different backend and front-end services. We will be launching it and running tests against it as part of this example.
 
-The [Pokeshop Demo App](/live-examples/pokeshop/overview) is a complete example of a distributed application using different backend and front-end services, implementation code is written in Typescript.
+The `docker-compose.yaml` file in the root directory of the quick start runs the Pokeshop Demo app, the OpenTelemetry Collector setup, and the [Tracetest Agent](/concepts/agent).
 
-The `docker-compose-yaml` file in the root directory is for the Pokeshop Demo app, the OpenTelemetry setup, and the [Tracetest Agent](/concepts/agent).
+The Typescript Quickstart has two primary files: a Typescript file `definitions.ts` that defines two Tracetest tests, and a Typescript file 'index.ts' that imports these test definitions and uses the "@tracetest/client" NPM package to run them multiple times. 
 
-### Docker Compose Network
+We will show you how to install the NPM package and use these two Typescript programs to programatically run Tracetest tests.
```
Comment (adnanrahic) [inline docs/docs/tools-and-integrations/typescript.mdx:99]:
```suggestion
We will show you how to install the NPM package and use these two TypeScript files to programmatically run Tracetest tests.
```

File: docs/docs/tools-and-integrations/typescript.mdx [hunk 11]
```diff
@@ -63,31 +63,50 @@ This is a simple quick-start guide on how to use the Tracetest `@tracetest/clien
...
 
-All `services` in the `docker-compose.yaml` are on the same network and will be reachable by hostname from within other services. For example, `jaeger:14250` in the `collector.config.yaml` file will map to the `jaeger` service, where port `14250` is the port where the Jaeger all-in-one instance accepts telemetry data.
+## Installing the `@tracetest/client` NPM Package
+
+The first step when using the Typescript NPM package is to install the `@tracetest/client` NPM Package. It is as easy as running the following command:
```
Comment (adnanrahic) [inline docs/docs/tools-and-integrations/typescript.mdx:103]:
```suggestion
The first step when using the TypeScript NPM package is to install the `@tracetest/client` NPM Package. It is as easy as running the following command:
```

File: docs/docs/tools-and-integrations/typescript.mdx [hunk 12]
```diff
@@ -261,76 +280,17 @@ const main = async () => {
...
-docker-compose up
+docker compose up -d
 ```
 
-You can visit the Pokeshop Demo UI at `http://localhost:8081/`.
-
 ## Finding the Results
 
-The output from the Typescript script should be visible in the terminal. Where you can find links to Tracetest to find details about trace-based test results.
+The output from the Typescript script should be visible in the log for the `tracetest-client` service via Docker Desktop. This log will show links to Tracetest for each of the test runs invoked by index.ts. Click a link to launch Tracetest and view the test result.
```
Comment (adnanrahic) [inline docs/docs/tools-and-integrations/typescript.mdx:293]:
```suggestion
The output from the TypeScript script should be visible in the log for the `tracetest-client` service in Docker Compose. This log will show links to Tracetest for each of the test runs invoked by `index.ts`. Click a link to launch Tracetest and view the test result.
```

**Review by adnanrahic — COMMENTED** [review]
Just some copywriting edits.

**Review by xoscar — APPROVED** [review]

**Pull request merged**